nominees/mbtkr.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "Mbtkr",
+  "aliases": [
+    ""
+  ],
+  "description": "An educational system for self taught people.",
+  "website": "http://mbtkr.rf.gd/",
+  "license": [
+    {
+      "spdx": "CC-BY-1.0",
+      "licenseURL": "http://mbtkr.rf.gd/course.php?ID=MATH101"
```

Comment:
Please check with submitter for a valid license URL that explicitly indicates the `CC-BY-1.0` license. Thanks.